react/components/Order/OrderActions.js

```diff
@@ -50,7 +49,7 @@ class OrderActions extends Component {
       paymentData: { transactions },
     } = order
 
-    const isOwner = email === this.state.loggedInEmail
+    const isOwner = email.startsWith(this.state.loggedInEmail)
```

Comment:
How sure are we that this new email field will contain the original value as a prefix? It would be good to get a confirmation from the OMS team (if you haven't done it yet, of course). Maybe Mourão can help confirm that, if needed (I couldn't mention him here, he's probably no within this vtex-apps org yet).